src/is_face_detector/faceDetection.py

```diff
@@ -6,17 +6,14 @@
 
 
 class FaceDetector:
-    def __init__(self, options):
-        self._op = options
-        self.faceCascade = cv2.CascadeClassifier(options['cascPath'])
-        self.log = Logger("FaceDetection", Logger.DEBUG)
+    def __init__(self):
+        self.faceCascade = cv2.CascadeClassifier('./src/is_face_detector/haarcascade_frontalface_default.xml')
     
     def detect(self, image):
         # Read the image
         self.image = image
         gray = cv2.cvtColor(self.image, cv2.COLOR_BGR2GRAY)
         self.faces = self.faceCascade.detectMultiScale(image=gray, scaleFactor=1.1, minNeighbors=1, minSize=(10,10), flags=cv2.CASCADE_SCALE_IMAGE)
```

Comment:
put these detector's parameters as constructor's parameters, and specify its on options file.
For default values, use get method, such as:
options.get('scaleFactor', 1.1)
In case of a not specified value, default will be used.

tip:
when defining detector configuration on options message, create a separate message to do so, and then add a field of it on options message.